src/personaclient/PersonaClient.php

```diff
@@ -265,6 +273,36 @@ public function isPresignedUrlValid($url,$secret) {
         return $valid;
     }
 
+    /**
+     * Get a user profile based off a gupid passed in
+     * @param string $gupid
+     * @param string $token
+     * @access public
+     * @return mixed
+     * @throws \InvalidArgumentException
+     * @throws \Exception
+     */
+    public function getUserByGupid($gupid, $token){
```

Comment:
Added to getUserByGuids - that only supports multiple users - so i'm only support a guids value that is an array.